nx-X11/config/cf/Imake.tmpl

```diff
@@ -759,6 +759,25 @@ TCLIBDIR = TclLibDir
 #define AlternateUsrLibDir YES
 #endif
 #endif
+
+#ifndef UsrDataDir
+#ifdef ProjectRoot
+#define UsrDataDir Concat4(ProjectRoot,/,share,/nx)
+#ifndef AlternateUsrDataDir
+#define AlternateUsrDataDir YES
+#endif
+#else
+#define UsrDataDir Concat4(/usr/local,/,share,/nx)
+#ifndef AlternateUsrDataDir
+#define AlternateUsrDataDir NO
+#endif
+#endif
+#else
+#ifndef AlternateUsrDataDir
+#define AlternateUsrDataDir YES
+#endif
+#endif
+
 #ifndef VarDirectory
```

Comment:
On  Mo 04 Jul 2016 22:40:16 CEST, Ulrich Sibiller wrote:

>> +#define UsrDataDir Concat4(ProjectRoot,/,share,/nx)
>> +#ifndef AlternateUsrDataDir
>> +#define AlternateUsrDataDir YES
>> +#endif
>> +#else
>> +#define UsrDataDir Concat4(/usr/local,/,share,/nx)
>> +#ifndef AlternateUsrDataDir
>> +#define AlternateUsrDataDir NO
>> +#endif
>> +#endif
>> +#else
>> +#ifndef AlternateUsrDataDir
>> +#define AlternateUsrDataDir YES
>> +#endif
>> +#endif
>> +
>>  #ifndef VarDirectory
>
> I think some indention would help reading this.
>
> ```
> #ifndef UsrDataDir
> #  ifdef ProjectRoot
> #    define UsrDataDir Concat4(ProjectRoot,/,share,/nx)
> #    ifndef AlternateUsrDataDir
> #      define AlternateUsrDataDir YES
> #    endif
> #  else
> #    define UsrDataDir Concat4(/usr/local,/,share,/nx)
> #    ifndef AlternateUsrDataDir
> #      define AlternateUsrDataDir NO
> #    endif
> #  endif
> #else
> #  ifndef AlternateUsrDataDir
> #    define AlternateUsrDataDir YES
> #  endif
> #endif
> ```

Looked at Imake.tmpl once more. The whole file could need indentation,  
but that is really beyond the scope of this PR.

Mike
-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de